src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -99,17 +119,28 @@ const NavBar: React.FC<NavBarPropsType> = ({ redirectionUrl }) => {
 
   const hasUpcomingEvents = futureUpcoming && futureUpcoming.length > 0;
 
+  const hasRadioStations = radioStations && radioStations.length;
+  const isSoundCloud =
+    !!hasRadioStations && RegExp("soundcloud").test(radioStations![0]);
+
   const sound = useMemo(
     () =>
-      radioStations && radioStations.length
+      radioStations && radioStations.length && !isSoundCloud
         ? new Audio(radioStations[0])
         : undefined,
-    [radioStations]
+    [isSoundCloud, radioStations]
   );
 
   const [isRadioPlaying, setIsRadioPlaying] = useState(false);
   const { volume, setVolume } = useRadio(isRadioPlaying, sound);
 
+  const soundcloudIframeRef = useRef<HTMLIFrameElement>(null);
+
+  // Append script to <body>
+  useEffect(() => {
+    loadScript("https://w.soundcloud.com/player/api.js");
+  }, []);
```

Comment:
Should we only do this when `isSoundCloud`?

Reply:
Good point